### PR TITLE
Add paths-ignore in Github actions

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Add paths-ignore to execute WKF only when you change code about app

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,14 +1,19 @@
----
```
Comment (pnu-s) [inline .github/workflows/main.yml:1]: Why delete this line?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
       - v*
+    paths-ignore:
+      -'fastlane/**'
+      -'README.md'
   pull_request:
     branches:
       - master
+    paths-ignore:
+      -'fastlane/**'
+      -'README.md'
```
Comment (pnu-s) [inline .github/workflows/main.yml:16]: If we want to ignore this paths for pull request, why not.

But I would keep them for the master branch.

**Review by pnu-s — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (Jean-BaptisteC) [reply .github/workflows/main.yml:1]: This line is useless

Comment (pnu-s) [reply .github/workflows/main.yml:1]: I disagree with that.

It's a convention to start YML files with `---`, so I would not say it's "useless"

**Review by pnu-s — APPROVED** [review]

**Pull request merged**